.github/workflows/chromatic.yaml

```diff
@@ -0,0 +1,36 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: 'Chromatic'
+
+# Event for the workflow
+on:
+  push:
+    branches:
+      - master
+    tags:
```

Comment:
Do you want it for tags and commits, or just commits? It'll run twice for a tag then - once when you push the commit then again when you tag that commit.

Reply:
OK, it'll just do it twice so I wanted to confirm.